resources/code/lf-workshop-js.template

```diff
@@ -838,6 +716,22 @@
                 }
             }
         },
+        "MetadataBucket": {
+            "Type": "AWS::S3::Bucket",
+            "Properties": {
+                "BucketName": {
+                    "Fn::Join": [
+                        "-",
+                        [
+                            "lf-metadata-bucket",
```

Comment:
As discussed, we do not want multiple buckets. please use lf-scripts-bucket-<acct-id> and have proper prefixes for the metadata and notebooks. I am ok for now but want this to be fixed after next week (when you add your ADFS changes)

Reply:
Also remove the word "bucket" from the bucket name

Reply:
not sure why it was used even with datalake bucket

Reply:
Sure will re-factor this bucket creation after the workshop next week.